tests/vars/main.yml

```diff
@@ -2,3 +2,14 @@
 ---
 rstudio_connect_install:
   - r-base
+rstudio_connect_config:
+  R:
+    ExecutableSystemScanning: true
+  HTTP:
+    Listen: :{{ rstudio_connect_www_port }}
+  Authentication:
+    Provider: "password"
+  'RPackageRepository "CRAN"':
+    URL: https://packagemanager.rstudio.com/cran/__linux__/{{ ansible_lsb.codename | lower }}/latest
+  'RPackageRepository "RSPM"':
+    URL: https://packagemanager.rstudio.com/cran/__linux__/{{ ansible_lsb.codename | lower }}/latest
```

Comment:
arent they the same? Why we keep two different names for them?

Reply:
Yep, they are the same - to be honest I just copy pasted them from the defaults. I'll remove the second `RPackageRepository`